packages/code-display/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@looker/code-display",
+  "version": "0.0.1",
```

Comment:
I think this version is fine. But just FYI this package will stay at `0.0.n` until you commit a breaking change (e.g. `feat!:`, `fix!:`, etc) as both `fix` AND `feat` only ever increment the "patch" for pre-major packages.

And, if/when you want to graduate to say `0.1.0`, then you'll have to manually update dependents as release-please won't make that "pre-major dependency's breaking change bump"